src/mysql_conn.erl

```diff
@@ -437,8 +512,8 @@ code_change(_OldVsn, _State, _Extra) ->
 
 %% --- Helpers ---
 
-%% @doc Executes a prepared statement and returns {reply, Reply, NextState}.
-execute_stmt(Stmt, Args, FilterMap, Timeout,
+%% @doc Executes a prepared statement and returns {Reply, NewState}.
+do_execute_stmt(Stmt, Args, FilterMap, Timeout,
```

Comment:
Why the "do"? Is it intended as a prefix for all internal functions? I don't like it very much as it makes the names clumsier. :-)

Reply:
It's a habit, I guess =^^= I often do it like this in other projects, at least until the interface has settled down a little. This way, I can always export nicely named functions and don't have to think up new ones or rename things in case they are already taken by something internal.

In this case, IIRC, I did it to be more consistent with `do_query` (`query` seems to be a reserved word, though I have no idea what it is ever for).

Reply:
Query was a reserved word in Erlang until Luic removed the reservation in a PR some years ago. It wasn't used for anything. This was before I started this project and `mysql:query/2,3,4,5` exist indeed. :-)

Anyway, the only functions exported in this module are the gen_server behaviour callbacks, so no need to prefix with "do" in general, so at least don't rename existing functions like this one.

Reply:
Strange... I googled a bit, what I found says `query` was removed in OTP 17. But the travis tests for mysql-otp also use R16B03-1 and pass despite `mysql:query` being in there, so I guess it was removed even earlier.

Reply:
Look here: https://github.com/erlang/otp/pull/1525

Reply:
Ah, so looks like it was done in R16B02. Could have asked Loïc himself, come to think of it =^^=

Anyway, I renamed `do_execute_stmt` back to `execute_stmt`, and did the same for `do_query` and `do_named_prepare` while I was at it.